nat-lab/tests/utils/connection_tracker.py

```diff
@@ -40,6 +40,9 @@ class ConnectionTrackerConfig:
     limits: ConnectionLimits
     target: FiveTuple
 
+    def get_key(self) -> str:
```

Comment:
what's the purpose of this method, when you can simple read key?

Reply:
I am paranoid reading the values directly, I think it's too fragile due to type safety, though I cannot exactly tell if having a method is stronger in that regard, having a function also allows for finding call sites easily which I prefer a lot with dynamic typing

Reply:
I guess it's just a personal preference. :+1:

Reply:
Sort of yeah...